File: Examples/DotNET/SDK/CSharp/Hyperlink/GetAllHyperlinks.cs
using System;
using Aspose.Cloud;

namespace CSharp.Hyperlink
{
    class GetAllHyperlinks
    {
        public static void Run()
        {
            string inputfile = "doc-sample-hyperlink.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Get all Hyperlinks from a Word
            WordsHyperlinksResponse wordsHyperlinksResponse = Common.WordsService.WordsHyperlinks.ReadDocumentHyperlinksCommonInfo(inputfile, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have " + wordsHyperlinksResponse.Hyperlinks.HyperlinkList.Count + " hyperlinks");
        }
    }
}

File: Examples/DotNET/SDK/VisualBasic/Hyperlink/GetAllHyperlinks.vb
Imports Aspose.Cloud

Namespace Hyperlink
    Class GetAllHyperlinks
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample-hyperlink.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Get all Hyperlinks from a Word
            Dim wordsHyperlinksResponse As WordsHyperlinksResponse = Common.WordsService.WordsHyperlinks.ReadDocumentHyperlinksCommonInfo(inputfile, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have " + wordsHyperlinksResponse.Hyperlinks.HyperlinkList.Count + " hyperlinks")
        End Sub
    End Class
End Namespace